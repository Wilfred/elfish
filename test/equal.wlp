(defun main ()
  (print "(= 1 1): ")
  (print (if (= 1 1) "true\n" "false\n"))

  (print "(= 1 2): ")
  (print (if (= 1 2) "true\n" "false\n"))

  (print "(= \"a\" \"a\"): ")
  (print (if (= "a" "a") "true\n" "false\n"))

  (print "(= \"a\" \"b\"): ")
  (print (if (= "a" "b") "true\n" "false\n"))

  (print "(= 1 \"foo\"): ")
  (print (if (= 1 "foo") "true\n" "false\n")))
