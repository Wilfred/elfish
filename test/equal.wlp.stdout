(= 1 1): true
(= 1 2): false
(= "a" "a"): true
(= "a" "b"): false
(= 1 "foo"): false
